Update m365-campaigns-conditional-access.md (#3238)
MicrosoftDocs/microsoft-365-docs: microsoft-365/campaigns/m365-campaigns-conditional-access.md

## Set up baseline policies

> 1. Go to [Azure portal](https://portal.azure.com), and then navigate to **Azure Active Directory** \> **Conditional Access** to create a new **Policy**

Reviewer: ```suggestion
1. Go to the [Azure portal](https://portal.azure.com), and then navigate to **Azure Active Directory** \> **Conditional Access** to create a **new policy**.
```

> See the following specific instructions for each policy

Reviewer: ```suggestion
See the following specific instructions for each policy: <br>
```

>     - [Require MFA for admins](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-administrators)

Reviewer: ```suggestion
    - [Require MFA for admins](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-administrators) <br>
```

>     - [Require MFA for users](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-end-users)  

Reviewer: ```suggestion
    - [Require MFA for users](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-end-users) <br>
```

>     - [Block legacy authentication](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-legacy-auth)

Reviewer: ```suggestion
    - [Block legacy authentication](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-legacy-auth) <br>
```

    - [Require MFA for service management](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-baseline-protect-azure)
    
> [!NOTE]

> > The Preview Policies no longer exists and the customer will need to create their own policy.

Reviewer: ```suggestion
> Preview policies no longer exist and users will need to create their own policies.
```
